namui-animation-editor/src/graph_editor/mod.rs

```diff
@@ -0,0 +1,101 @@
+use namui::{prelude::*, types::*};
+use namui_prebuilt::table::*;
+mod graph_window;
+use crate::*;
+mod preview_window;
```

Comment:
```suggestion
mod graph_window;
mod preview_window;
use crate::*;
use namui::{prelude::*, types::*};
use namui_prebuilt::table::*;
```

This is a result of gouping and linting imports